rust/workspace/Cargo.toml

```diff
@@ -5,7 +5,3 @@ members = [
     "prql",
 ]
 resolver = "2"
-
-[patch.crates-io]
-# Ref: https://github.com/lotabout/tuikit/pull/51
-tuikit = { git = "https://github.com/azat-rust/tuikit.git", rev = "e1994c0e03ff02c49cf1471f0cc3cbf185ce0104" }
```

Comment:
This requires https://github.com/skim-rs/skim/pull/783

Reply:
Yep, thank you for reopening.